libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/PredictiveCacheController.kt

```diff
@@ -174,4 +162,13 @@ class PredictiveCacheController @JvmOverloads constructor(
         Log.e(TAG, error)
         predictiveCacheControllerErrorHandler?.onError(error)
     }
+
+    private fun MapboxMap.tileStore(): TileStore {
+        val tileStorePath = this.getResourceOptions().tileStorePath
+        return if (tileStorePath == null) {
+            TileStore.getInstance()
+        } else {
+            TileStore.getInstance(tileStorePath)
+        }
+    }
```

Comment:
How can we make this function as robust to use as possible?

The map and navigation tilestore instances should always have the same path. Should we add logs if that's not the case? Throw exceptions?

Reply:
removed this changes, rebase issue